pkg/data/values.go

```diff
@@ -18,18 +22,38 @@ func GetValueN(data map[string]interface{}, keys ...string) interface{} {
 	return val
 }
 
-func GetValue(data map[string]interface{}, keys ...string) (interface{}, bool) {
+func GetValue(data interface{}, keys ...string) (interface{}, bool) {
```

Comment:
I don't love this addition of optionally taking a slice and a number key as a string. Can we just make a new function for slices?

Reply:
I could revert this function to only take a map and to call out to another function when it encounters a nested slice, that covers the use case I care about. But I can't completely separate the map and slice functions or have it not take a number as a string, because the data is unstructured and the caller (steve) doesn't know whether it is going to encounter a slice.

Reply:
Resolving, decision made offline to allow a slice and a string index value.